packages/config/config/devices/0x028c/pwly-7828-a1.json

```
@@ -0,0 +1,287 @@
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 255,
+			"defaultValue": 0,
+			"unit": "°F",
+			"unsigned": true,
+			"options": [
+				{
+					"label": "No change",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "6",
+			"label": "LED Dimming",
+			"description": "When powered via C-Wire",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 3,
+			"defaultValue": 3,
+			"options": [
+				{
+					"label": "Low",
+					"value": 0
+				},
+				{
+					"label": "Medium",
+					"value": 1
+				},
+				{
+					"label": "High",
+					"value": 2
+				},
+				{
+					"label": "Auto",
+					"value": 3
+				}
+			]
+		},
+		{
+			"#": "7",
+			"label": "Demand Response Mode",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 0,
+			"options": [
+				{
+					"label": "Off",
+					"value": 0
+				},
+				{
+					"label": "On",
+					"value": 1
+				}
+			]
+		},
+		{
+			"#": "8",
+			"label": "Swing Setting",
+			"description": "Number of degrees from the set point at which the HVAC will turn on",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 10,
+			"defaultValue": 1,
+			"unit": "°F",
+			"options": [
+				{
+					"label": "No swing",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "9",
+			"label": "Differential Setting",
+			"description": "Number of degrees from the set point at which the 2nd stage will engage",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 10,
+			"defaultValue": 3,
+			"unit": "°F",
+			"options": [
+				{
+					"label": "No differential",
+					"value": 0
+				}
+			]
+		},
+		{
+			"#": "10",
+			"label": "OTA (Over The Air) programming error code",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 10,
+			"defaultValue": 0,
+			"readOnly": true,
+			"options": [
+				{
+					"label": "No error",
+					"value": 0
+				},
+				{
+					"label": "Battery low, reflashing not possible",
+					"value": 1
+				},
+				{
+					"label": "OTA Cancelled before completion",
+					"value": 2
+				},
+				{
+					"label": "Fragment CRC error",
+					"value": 3
+				},
+				{
+					"label": "Total image CRC error",
+					"value": 4
+				},
+				{
+					"label": "External flash error",
+					"value": 5
+				},
+				{
+					"label": "Fragment received out of order",
+					"value": 6
+				},
+				{
+					"label": "Invalid firmware or manufacture ID",
+					"value": 7
+				},
+				{
+					"label": "OTA session not properly initialized",
+					"value": 8
+				},
+				{
+					"label": "Same image currently running",
+					"value": 9
+				},
+				{
+					"label": "Time out during OTA download between fragments",
+					"value": 10
+				}
+			]
+		},
+		{
+			"#": "11",
+			"label": "Display always on",
+			"valueSize": 1,
+			"minValue": 0,
+			"maxValue": 1,
+			"defaultValue": 1,
+			"options": [
+				{
+					"label": "Display off in C-Wire powered mode",
+					"value": 0
+				},
+				{
+					"label": "Display on in C-Wire powered mode",
+					"value": 1
+				}
```

We like to use Enable/Disable for binary options. E.g. like this:
```suggestion
			"label": "Display On in C-Wire Powered Mode",
			"valueSize": 1,
			"minValue": 0,
			"maxValue": 1,
			"defaultValue": 1,
			"options": [
				{
					"label": "Disable",
					"value": 0
				},
				{
					"label": "Enable",
					"value": 1
				}
```